civiclens/utils/database_access.py

```diff
@@ -65,3 +71,53 @@ def pull_data(
         results = pl.DataFrame(results, schema=schema)
 
     return results
+
+
+def upload_comments(connection: Database, comments: RepComments) -> None:
+    """
+    Uploads comment data to database.
+
+    Inputs:
+        connection: Postgres client
+        comments: comments to be uploaded
+    """
+    query = """INSERT INTO regulations_nlpoutput (
+                    "id",
+                    "rep_comments",
+                    "doc_plain_english_title",
+                    "num_total_comments",
+                    "num_unique_comments",
+                    "num_representative_comment",
+                    "topics",
+                    "num_topics",
+                    "last_updated",
+                    "document_id"
+                    ) VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s)
+                    ON CONFLICT (id) DO NOTHING;
+                    """
+
+    values = (
+        comments.uuid,
+        json.dumps(comments.rep_comments),
+        comments.doc_plain_english_title,
+        comments.num_total_comments,
+        comments.num_unique_comments,
+        comments.num_representative_comment,
+        json.dumps(comments.topics),
+        len(comments.topics),
+        comments.last_updated.strftime("%m/%d/%Y, %H:%M:%S"),
+        comments.document_id,
+    )
+
+    try:
+        cursor = connection.cursor()
+        cursor.execute(query, values)
+        connection.commit()
+
+    except Exception as e:
```

Comment:
Could be a backlog thing for now but logging failed update id's would be helpful. Also, it feels very possible there is junk data that could screw up the types for inserting to the database as well. In that case we could consider saving the values for problematic fields to help us clean data better if the errors we log aren't verbose enough.